runtime/doc/news.txt

```diff
@@ -116,6 +116,9 @@ The following changes may require adaptations in user config or plugins.
   upstream tree-sitter and Helix to make it easier to share queries. The full
   list is documented in |treesitter-highlight-groups|.
 
+• |:TOhtml| has been totally rewritten in Lua. Many legacy and (what we deem)
+  unnecessary options have been removed and some of the less used features
+  have been removed. Some features were adder like treesitter support.
```

Comment:
```suggestion
  have been removed. Some Nvim-specific features were added, such as treesitter support.
```
(with linewrap & whitespace changes)

Reply:
I think this text can be significantly shortened; no need to give rationale here.